.config/theme_overrides/404.html

```diff
@@ -0,0 +1,29 @@
+<!--
+  Copyright (c) 2016-2023 Martin Donath <martin.donath@squidfunk.com>
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to
+  deal in the Software without restriction, including without limitation the
+  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+  sell copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+  IN THE SOFTWARE.
+-->
+
+{% extends "main.html" %}
+
+<!-- Content -->
+{% block content %}
+<h1>We looked, but couldn’t quite find that page.</h1>
+<h3>
+  The page you are looking for has been removed, is temporarily unavailable or
+  does not exist. Maybe try the address again, or head to the homepage.
+</h3>
```

Comment:
Preview https://civicactions-handbook--1043.org.readthedocs.build/en/1043/not-found

I think having a header for all that text is not good practice for accessibility. Are you looking to make the text larger?

Reply:
@dmundra Yes.. I was just hoping to bump up the text size.

Reply:
.... but, it's okay if it has to be the standard body size.

Reply:
You can wrap that in a div and give it a unique class. Then CSS can be added to this file https://github.com/CivicActions/guidebook/blob/new-404-page/assets/css/extra.css to style it. Do you want to try that?

Reply:
Thanks for CSS guidance!  Will try that!